tests/ttnn/unit_tests/operations/test_uniform.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import time
+import pytest
+import numpy as np
+import ttnn
+from collections import Counter
+from loguru import logger
+from tests.ttnn.unit_tests.operations.test_utils import (
+    get_compute_kernel_options,
+    compute_kernel_options,
+    compute_kernel_ids,
+    get_lib_dtype,
+)
+from models.utility_functions import skip_for_grayskull
+from enum import Enum
+
+
+class TestMode(Enum):
+    VALIDATE = 0
+    BENCHMARK = 1
+
+
+def check_torch_uniform_bfloat16():
+    input = torch.zeros(10, 10, dtype=torch.bfloat16).uniform_(2.1, 2.11)
+    logger.info(input)
+
+
+# ttnn is faster than torch if the tensor is big enough: [1024, 1024]
```

Comment:
This comment will need to be updated when the PR is done, I think the final version will be faster.

Reply:
I've updated the comment.